pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -765,64 +765,98 @@ func TestGetTaskFunc_VerifySuccess(t *testing.T) {
 	resolvedMatched := test.NewResolvedResource(signedTaskBytes, nil, matchPolicyRefSource, nil)
 	requesterMatched := test.NewRequester(resolvedMatched, nil)
 
+	warnPolicyRefSource := &v1beta1.RefSource{
+		URI: "	warnVP",
+	}
+	resolvedUnsignedMatched := test.NewResolvedResource(unsignedTaskBytes, nil, warnPolicyRefSource, nil)
+	requesterUnsignedMatched := test.NewRequester(resolvedUnsignedMatched, nil)
+
 	taskRef := &v1beta1.TaskRef{ResolverRef: v1beta1.ResolverRef{Resolver: "git"}}
 
+	// the test covers those cases:
+	// VerificationPass: 1) 2) 3) signed task passes when no-match-policy is set to fail, warn, ignore
+	// VerificationWarn: 4) 5) 6) warn no-match-policy and failed warn policy
+	// VerificationSkip: 7) 8) ignore no match policies
```

Comment:
It looks like this comment may already not be in sync with the test cases-- I think it may lead to confusion

Reply:
To clarify, I suggest removing the comment, and letting test case names serve as documentation, because the comment can easily become out of sync and cause confusion.